commit 54a0b4a5d0857e60489579a02240d691d3b71f9d
Author: Paul McCarthy <pauldmccarthy@gmail.com>
Date:   Tue Jul 22 18:09:55 2025 +0100

    BF: Use WIN32_MSVC_NAMING and not wxUSE_MSVC_NAMING

diff --git a/build/cmake/functions.cmake b/build/cmake/functions.cmake
index 643ba780..8a81a80a 100644
--- a/build/cmake/functions.cmake
+++ b/build/cmake/functions.cmake
@@ -482,7 +482,7 @@ macro(wx_add_library name)
         set_target_properties(${name} PROPERTIES PROJECT_LABEL ${name_short})
 
         # Setup install
-        if(wxUSE_MSVC_NAMING)
+        if(WIN32_MSVC_NAMING)
             set(runtime_default_dir "lib")
         else()
             # configure puts the .dll in the bin directory
